### ISSUE TITLE
Low loss but bad result

### ISSUE THREAD

**Issue by JingyuanHu** [1adrianb/face-alignment#107]
Hi @1adrianb , thank you for your great work, it is really cool. 
However, I met some problems when I try to reproduce 2d-fan result by pytorch.  I wrote my training process inspired by  2d-fan code in lua version . [https://github.com/1adrianb/2D-and-3D-face-alignment](url)
And my training loss is MSE.  My loss is decreased to around 400 with a bad performance. However i found the model you provided have a very good performance with MSE loss is around 1000.
It is strange, right? Maybe you can give me some advice. Is MSE suitable in this problem?

**Comment by JingyuanHu** [comment]
Sorry , It is my problem. I generate a wrong label. This issue can be closed
